packages/app/src/components/nominationInfo/healthProviderSearch/index.js

```diff
@@ -0,0 +1,70 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { SearchResultDataContext } from '../../../utils/context/SearchResultsContext';
+import { NominationsDataContext } from '../../../utils/context/NominationsContext';
+import styles from './styles.module.css';
+import { faCaretSquareDown } from '@fortawesome/free-solid-svg-icons';
+
+const SearchHealthProvider = () => {
+  const [SearchHealthcareProvider, setSearchHealthcareProvider] = useContext(SearchResultDataContext);
+  const [NominationsData, setNominationsData] = useContext(NominationsDataContext);
+
+  const findSearchResults = (searchTerm) => {
+    let filteredNoms = [];
+    if (NominationsData && searchTerm !== undefined) {
+      filteredNoms = NominationsData.filter((nomination) => {
+        return [
+          nomination.providerName,
+          nomination.patientName,
+          nomination.nominationName,
+          nomination.representativeName,
+        ].some((nom) => nom.includes(searchTerm));
+      });
+      setSearchHealthcareProvider(filteredNoms);
+    }
+  };
+
+  const { id } = useParams();
+
+  useEffect(() => {
+    setTimeout(function () {
+      findSearchResults(id);
+    }, 1000);
```

Comment:
What's the reason for delaying for 1 second here?

Reply:
I resolved this issue. This should be okay now.